mlir/include/mlir/Dialect/Vector/IR/VectorOps.td

```diff
@@ -10,9 +10,13 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef VECTOR_OPS
-#define VECTOR_OPS
+#ifndef MLIR_DIALECT_VECTOR_IR_VECTOR_OPS
+#define MLIR_DIALECT_VECTOR_IR_VECTOR_OPS
 
+include "Vector.td"
+include "VectorAttributes.td"
```

Comment:
I think these are usually prefixed with the full path although not strictly necessary (e.g. in ArithOps.td)
eg
include "mlir/Dialect/Vector/IR/Vector.td"
...

Reply:
done